system-addon/data/content/tippytop/top_sites.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "title": "amazon",
+    "urls": ["https://www.amazon.com/", "https://www.amazon.ca/", "https://www.amazon.de/", "https://www.amazon.co.uk/", "https://www.amazon.fr/"],
+    "image_url": "amazon-com@2x.png"
+  },
+  {
+    "title": "facebook",
+    "url": "https://www.facebook.com/",
+    "image_url": "facebook-com@2x.png"
```

Comment:
Not sure what we'll do for other languages that use facebook.com, but the screenshot is english.